docs-en/chapter_computational_complexity/summary.md

```diff
@@ -2,48 +2,48 @@
+- Complexity analysis overcomes the drawbacks of real-world testing. The analysis results can apply to all operating platforms and reveal the algorithm's efficiency under variant data scales.
 
-**Time complexity**
+**Time Complexity**
 
-- Time complexity is used to measure the trend of the algorithm's running time with the growth of data volume, which can effectively evaluate the efficiency of the algorithm, but it may fail in some cases, such as when the input data volume is small or the time complexity is the same, it is not possible to accurately compare the advantages and disadvantages of the efficiency of the algorithm.
-- The worst time complexity is denoted by a large $O$ symbol, which corresponds to the asymptotic upper bound of the function, reflecting the level of growth in the number of operations $T(n)$ as $n$ tends to positive infinity.
-- Extrapolating the time complexity is done in two steps, first counting the number of operations and then determining the asymptotic upper bound.
-- Common time complexities are $O(1)$, $O(\log n)$, $O(n)$, $O(n \log n)$, $O(n^2)$, $O(2^n)$, and $O(n!)$, etc., in order of their degree.
-- The time complexity of certain algorithms is non-fixed and is related to the distribution of the input data. The time complexity is categorized into worst, best, and average time complexity, with the best time complexity hardly ever being used because the input data generally needs to satisfy stringent conditions to achieve the best case.
-- The average time complexity reflects the efficiency of an algorithm's operation under random data inputs, and is closest to the performance of algorithms in real-world applications. Calculating the average time complexity requires a statistical input data distribution as well as a synthesized mathematical expectation.
+- Time complexity is used to measure the trend of algorithm running time as the data size grows., which can effectively evaluate the algorithm's efficiency. However, it may fail in some cases, such as when the input volume is small or the time complexities are similar, making it difficult to precisely compare the efficiency of algorithms.
+- The worst time complexity is denoted by big $O$ notation, which corresponds to the asymptotic upper bound of the function, reflecting the growth rate in the number of operations $T(n)$ as $n$ tends to positive infinity.
+- The estimation of time complexity involves two steps: first, counting the number of operations, and then determining the asymptotic upper bound.
+- Common time complexities, from lowest to highest, are $O(1)$, $O(\log n)$, $O(n)$, $O(n \log n)$, $O(n^2)$, $O(2^n)$, and $O(n!)$.
+- The time complexity of certain algorithms is not fixed and depends on the distribution of the input data. The time complexity can be categorized into worst, best, and average. The best time complexity is rarely used because the input data must meet strict conditions to achieve the best-case.
```

Comment:
The time complexity can be categorized into **worst-case, best-case**, and average. The **best-case** time complexity is rarely used ...